lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -129,6 +130,43 @@ def configure(conf)
       log_level = conf['@log_level'] || conf['log_level']
       log.warn "Consider to specify log_level with @log_level." unless log_level
     end
+
+    if @dlq_handler && @dlq_handler.is_a?(Hash)
```

Comment:
These added lines are bit of long to write in `#configure`.
Could you extract as a method like as `configure_dlq_handler`?